_dashboards/management/advanced-settings.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Advanced settings
+parent: Dashboards Management
+nav_order: 40
+---
+
+# Advanced settings
+
+Use the **Advanced settings** page to modify settings that govern OpenSearch Dashboards behavior. For example, you can use the page to enable or disable the dark mode, change the default language, change the default number of results displayed on a dashboard, or change the default time zone. See the following sections to learn more about the specific settings.
```

Comment:
I think the introduction should explain a bit more about conceptually what types of settings and configurations are accessible/editable here, and how they differ from other configuration/options/management UIs in the application. For example, any OpenSearch Dashboard plugin can register additional settings to any of the existing categories, or even register a new category of settings.

The other primary source of settings and configurations is the `opensearch_dashboards.yml` configuration file (which we appear to have no documentation page for - there should be an OpenSearch Dashboards equivalent of https://opensearch.org/docs/2.9/install-and-configure/configuration/). Some settings can actually be defined in either place, and admins need to understand the precedence between the two, and the key difference that Advanced Settings can be updated at any time, whereas config values are applied at startup and require a redeploy/restart.

Another key perspective is that, with the Security plugin enabled, advanced settings apply to the currently selected tenant. So you can make whatever changes you want in your private tenant, but any changes made to the global tenant apply to all users of that tenant.

Reply:
I mostly agree with Josh here but my only callouts here are:

1. Dont list the specific settings as a definitive list, we can have more that arent listed that another plugin has added but we arent aware of.
2. Have a link to the configuration page highlighting that some settings are controlled by the config and may not be available via advanced settings, and some config settings can be overriden by the advanced settings.